src/Symfony/Component/Debug/Tests/Fixtures/OutsideInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Test\Symfony\Component\Debug\Tests\Fixtures;
+
+interface OutsideInterface
+{
+    /**
+     * @return void
```

Comment:
fabbot wants to remove the return void, but they are useful for tests. So I guess the files thats contain it needs to be ignored.